speech/cloud-client/quickstart.py

```diff
@@ -46,7 +46,8 @@ def run_quickstart():
     config = types.RecognitionConfig(
         encoding=enums.RecognitionConfig.AudioEncoding.LINEAR16,
         sample_rate_hertz=16000,
-        language_code='en-US')
+        language_code='en-US',
+        enable_word_time_offsets=False)
```

Comment:
Agreeing with @jonparrott on this:  This is definitely something we want to make clear in the API/client library reference, but adding this in the sample codes would make the API seem harder to use than it actually is (note that the other parameters `encoding`, `langauge_code`, and `sample_rate_hertz` are all required, but `enable_word_time_offsets` is optional and defaults to false).

What are some possible use case scenarios where not showing `enable_word_time_offsets=False` could cause friction for the users?